src/applications/mhv-medical-records/util/helpers.js

```diff
@@ -321,8 +318,7 @@ export const parseDate = str => {
       'November',
       'December',
     ];
-
-    const monthName = monthNames[month + 1];
+    const monthName = monthNames[month - 1];
```

Comment:
Could this function use a date-fns formatter, rather than maintaining a separate list of month names?